internal/interceptor/server.go

```diff
@@ -464,6 +468,45 @@ func renderPipeline(odsConfig *config.ODS, data PipelineData) ([]byte, error) {
 			},
 		},
 	})
+	if enableAggregateTaskStatus {
```

Comment:
I don't believe we need a separate task. The only change we need to make is to pass the parameter `aggregateTasksStatus` to the `ods-finish` task above ...

Reply:
Done